src/lib/SX1280Driver/SX1280.cpp

```diff
@@ -363,5 +363,5 @@ void ICACHE_RAM_ATTR SX1280Driver::GetLastPacketStats()
 
     hal.ReadCommand(SX1280_RADIO_GET_PACKETSTATUS, status, 2);
     instance->LastPacketRSSI = -(int8_t)(status[0] / 2);
-    instance->LastPacketSNR = (int8_t)(status[1] / 4);
+    instance->LastPacketSNR = (int8_t)status[1] / 4;
```

Comment:
Because both these values are int8_t you can completely remove the casting.
i.e.
```
instance->LastPacketRSSI = -(status[0] / 2);
instance->LastPacketSNR = status[1] / 4;
```

Reply:
I have to disagree that. It might give hint to compiler... I think it won't work like that. 
let's take an example:
```
    uint8_t _buff[2] = {(uint8_t)-20, (uint8_t)-40};
    temp = _buff[1] / 4;
    printf("temp: %d\n", temp);
```
here output will be `temp: 54`.

and if I cast it:
```
    temp = (int8_t)_buff[1] / 4;
    printf("temp: %d\n", temp);
```
output will be `temp: -10`.

So it is better to cast it explicitly even the compiler detects it correctly.

Reply:
I see what you are saying. So the `status` buffer is actually bytes that are interpreted as `int8_t` in this case rather than `uint8_t`. Would it be better if the buffers were declared as `byte` rather than `uint8_t` to make that intention clear? Which would mean that methods like `ReadCommand` and `ReadRegister` should take `byte` pointers.

Reply:
`typedef uint8_t byte ;` so the `byte` does not make a difference here :) 
that casting is handled at compile time so no need to worry about performance. I would keep code with castings ;)

Reply:
I know it won't make difference to the code, but it will to the humans reading the code. It's helps to describe the intent.
Keep the casts, you need them for sure.